Classes/Repository/SysLanguageRepository.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+
+namespace DirectMailTeam\DirectMail\Repository;
+
+class SysLanguageRepository extends MainRepository {
+    protected string $table = 'sys_language';
+    
+    /**
+     * @return array|bool
+     */
+    public function selectSysLanguageForSelectCategories(string $lang, $sys_language, $static_languages) //: array|bool
+    {
+        $queryBuilder = $this->getQueryBuilder($this->table);
+
+        return $queryBuilder
+            ->select('sys_language.uid')
+            ->from($this->table)
+            ->leftJoin(
+                $this->table,
+                'static_languages',
+                'static_languages',
+                $queryBuilder->expr()->eq('sys_language.language_isocode', $queryBuilder->quoteIdentifier('static_languages.lg_typo3'))
+            )
+            ->where(
+                $queryBuilder->expr()->eq('static_languages.lg_typo3', $queryBuilder->createNamedParameter($lang . $sys_language . $static_languages))
```

Comment:
I got an SQL error

````
An error occurred trying to process items for field "Subscribe to categories" (An exception occurred while executing 'SELECT `sys_language`.`uid` FROM `sys_language` LEFT JOIN `static_languages` `static_languages` ON (`sys_language`.`language_isocode` = `static_languages`.`lg_typo3`) AND (`static_languages`.`deleted` = 0) WHERE (`static_languages`.`lg_typo3` = ?) AND (`sys_language`.`hidden` = 0)' with params ["en AND `static_languages`.`deleted` = 0"]: Illegal mix of collations (utf8_general_ci,IMPLICIT) and (utf8_unicode_ci,IMPLICIT) for operation '=').
````

Reply:
I think the problem is, that `static_languages` is using `utf_unicode_ci` and `sys_language` `utf8_general_ci` collation.

Reply:
is local problem on my dev.